src/samples/EvalClient/EvalClient.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net9.0</TargetFramework>
```

Comment:
The other samples in the repo are .net80. It might be useful to stay coordinated to avoid version issues,

Reply:
Oh, didn't notice, good point!!